tests/std/tests/VSO_0157762_feature_test_macros/test.compile.pass.cpp

```diff
@@ -258,6 +258,20 @@ STATIC_ASSERT(__cpp_lib_barrier == 201907L);
 #endif
 #endif
 
+#if _HAS_CXX23

```

Comment:
you also have changed `__cpp_lib_ranges ` to 202202L so you need to correct the test.

https://github.com/microsoft/STL/blob/57026a2f047c8dbda02d4f30db2280e942342545/tests/std/tests/VSO_0157762_feature_test_macros/test.compile.pass.cpp#L1355-L1367

Reply:
Thanks for that hint! I just fixed that in https://github.com/microsoft/STL/pull/2661/commits/fe9aaf231214dc58eac9b5ecffcf5ad9baa6eefa . I followed the scheme from `__cpp_lib_optional` and used `!defined(__EDG__)` only in the first `if`-condition. Is that correct?

Reply:
Sorry, I had an issue with my ISP  so I couldn't reply earlier.
I added some suggestions and I tested locally that test on your branch with my suggestions.
```
python tests\utils\stl-lit\stl-lit.py ..\..\..\tests\std\tests\VSO_0157762_feature_test_macros -v
```